src/App.tsx

```diff
@@ -1,61 +1,145 @@
+// import React, { useState } from 'react';
 import './App.scss';
+import { useEffect, useState } from 'react';
+import { Todo } from './Types/Todo';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+import { User } from './Types/User';
+import { TodoList } from './components/TodoList';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+function getUser(userId: number): User | null {
+  const foundUser = usersFromServer.find(user => user.id === userId);
+
+  return foundUser || null;
+}
+
+export const App: React.FC = () => {
+  const visibleUsers = [...usersFromServer];
+  const [title, setTitle] = useState('');
+  const [selectedUser, setSelectedUser] = useState(0);
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [completed, setCompleted] = useState(false);
+  const [tittleError, setTittleError] = useState(false);
+  const [selectedError, setSelectedError] = useState(false);
+
+  useEffect(() => {
+    setTodos(todosFromServer.map(todo => ({
+      ...todo,
+      user: getUser(todo.userId),
+    })));
+  }, []);
```

Comment:
You don't need useEffect since the the data is not coming from a real server. You can just prepare it outside of the component and set the initial state.

Reply:
agree with @ysrckr